tools/cldr/cldr-to-icu/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.unicode.icu</groupId>
+    <artifactId>cldr-to-icu</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.5.1</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+
+    <repositories>
+        <repository>
+            <id>local-maven-repo</id>
```

Comment:
like to avoid this, see other comments.

Reply:
So would I, but right now it's the only way to make running this vaguely hermetic. Yes, we should redo the CLDR build stuff, but that's several weeks of work.

Reply:
https://github.com/unicode-org/icu/pull/765 is a draft towards some of that several weeks.

Reply:
So that seems to only be redoing things to push the JAR to Maven after building with Ant.
For making it build with Maven (which is what you'd want to get rid of some of the really messy dependency issues) you need a fairly non-trivial refactoring of lots of code in and around the "util" package. Right now "util" has circular deps with lots of other classes in other packages, which need to be broken if we don't want a monolithic JAR for the API that contains 10M of unused code.

Reply:
> So that seems to only be redoing things to push the JAR to Maven after building with Ant.

For the ICU `utilities.jar` side — yes.

Reply:
> push the JAR to Maven after building with Ant

… which is how ICU's maven deployments work

Reply:
Right, but that won't fix the cyclic deps and non-hermetic/non-incremental building that Ant has (Ant is only reliable if you previously did a "clean" before a build since it relies on time stamps rather than content hashing to determine "freshness"). I'd be interested in being given the chance to move CLDR to build with Maven.

Reply:
> Right, but that won't fix the cyclic deps and non-hermetic/non-incremental building that Ant has

it's meant as an incremental step

>  I'd be interested in being given the chance to move CLDR to build with Maven.

Right. Perhaps we can collaborate.